modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -0,0 +1,112 @@
+
+* You must use xref:install-rtfctl.adoc[Runtime Fabric command line utility (`rtfctl`)] version 0.3.150 or higher.
+
+* If you create a Runtime Fabric to use with a local registry, you cannot later reconfigure it to use the `rtf-runtime-registry` endpoint. To use the `rtf-runtime-registry` endpoint, you must install a new Runtime Fabric using the standard installation procedure.
+
+* You're responsible for synchronization between your registry and the Runtime Fabric registry. MuleSoft doesn't provide additional software to synchronize images between the registries.
+
+* If you use a local registry that needs authentication, synchronization and propagation of a pull-secret between different namespaces is your responsibility. Alternatively, Runtime Fabric can synchronize your pull-secret across different namespaces if your secret uses the following label:
+
+----
+  labels:
+    rtf.mulesoft.com/synchronized: "true"
+----
+
+* If you need to authenticate access to your local registry, ensure the https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials[corresponding credentials are configured as a Kubernetes secret in the `rtf-` namespace^]. You'll need the name of the configured secret when you install Runtime Fabric. 
+
+* You also need to synchronize in your registry the Mule runtime versions you intend to use for application deployments. Refer to the xref:release-notes::runtime-fabric/runtime-fabric-runtimes-release-notes.adoc[Runtime Fabric Patch Update Release Notes] for information about the latest Mule runtime patch versions. 
+
+== Configure a Local Registry for Use with Runtime Fabric 
+
+To configure a local registry you must gather and add the necessary credentials to synchronize the registries.
+
+=== Before You Begin
+
+Ensure you've performed the following tasks: 
+
+. Setup, configure, and test a local Docker image registry.
+. Synchronize to your local registry all the Docker images you need to install Runtime Fabric:
++
+[NOTE]
+Dependency versions are specific to the Runtime Fabric version.
+ 
++
+* appInit: `mulesoft/rtf-app-init:v1.0.35`
+* muleClusterIpService: `mulesoft/rtf-mule-clusterip-service:v1.2.36`
+* resourceFetcher: `mulesoft/rtf-resource-fetcher:v1.0.46`
+* nginx: `mulesoft/base-image-nginx-1.21.1:v1.1.9`
+* monitoringSidecar: `mulesoft/dias-anypoint-monitoring-sidecar:v1.3.18`
+* clusterOps: `mulesoft/rtf-cluster-ops:v1.1.25`
+* securityFabricEdge: `mulesoft/securityfabric-edge:v1.1.346`
+* externalLogForwarder: `mulesoft/rtf-pkg-fluentbit:v1.2.53`
+* coreAction: `mulesoft/rtf-core-actions:v1.0.11`
+* persistenceGateway: `mulesoft/rtf-object-store:v1.0.44`
+* rtfDaemon: `mulesoft/rtf-daemon:v1.0.3`
+
```

Comment:
This is where we need to transition from background info to the actual task, which gets lost being included with the Before You Begin narrative.